doc/iterations/2024.3/shapes/confirmation_dialog_on_delete.adoc

```diff
@@ -0,0 +1,52 @@
+= Confirmation dialog on delete
+
+== Problem
+
+Sirius Web does not have an undo/redo mechanism.
+When an element has been deleted, either from the Explorer view, from any representation, or any other view, there is no way to retrieve it.
+All delete actions are executable in one click.
+Any accidental click on a delete action can lead to the unwanted loss of data.
+
+== Key Result
+
+No delete action should be done in one click.
+
+== Solution
+
+A confirmation dialog should be displayed when a delete action (click on the delete tool on a node, click on delete key on a node, click on the delete menu item on an object in the Explorer view) has been called.
```

Comment:
Also when you use the DEL key in a diagram.

Reply:
Yes it is written in the paragraph: "click on delete key on a node". I will update the message to "click on DEL key on a node" to be more explicit.